.circleci/config.yml

```diff
@@ -679,15 +679,17 @@ jobs:
       - attach_workspace:
           at: *root_path
       - setup-shared-env
+      # Deploy Housekeeping first so hopefully liquibase migrations
+      # are ran before DDP boots up.
```

Comment:
We should start trying to make deployments forward and backwards compatible as far as schema changes go--if we want zero downtime, there will be a window where the schema changes but an "older" version of code will be deployed.